includes/init_includes/init_templates.php

```diff
@@ -33,19 +33,14 @@
  * Allow admins to switch templates using &t= URL parameter
  */
 if (zen_is_whitelisted_admin_ip()) {
-    $templates = array();
-    foreach($result as $row) {
-        $templates[] = $row['template_dir'];
-    }
```

Comment:
Excluding the check of this array means the only check for templates is against the filesystem.

I intended it to require that the template be registered in the admin in order for catalog-side to be able to switch to it.

Reply:
This seems like a really confusing and hard to figure out additional step.  We already have the protection that only admins can do this - why not make it easy for them?

Reply:
I'll concede.

Please also remove `and that it matches an available choice` from the comment.

And change the commit and PR title from "Fix..." to "Simplify..." or "Change...".

Reply:
Ugh sorry for the double commit - please squash.

Reply:
I'd hoped you would have updated the commit title as well (part of the amend+forcepush), not just the PR title.